ocaml/xapi/server_helpers.ml

```diff
@@ -75,6 +75,8 @@ let exec ?marshaller ?f_forward ~__context f =
           (* use the forwarding layer (NB this might make a local call ultimately) *)
           forward ~local_fn:f ~__context
     in
+    (* For forwarded task, we should not complete it here, the server which forward the task will complete it *)
+    if not (Context.forwarded_task __context) then
```

Comment:
This code has become dependent on a condition but indentation has not changed to reflect this. Please check the indentation of the code.